documentation/github-project-guidelines.md

```diff
@@ -0,0 +1,48 @@
+﻿# Guidelines for our GitHub projects
+
+In this place we will have the documentation for how to handle issues
+among projects of **TecnoYucas**.
+
+## General notes
+
+### Some rules first about the project
```

Comment:
I think it would be better if this was something like `General rules for projects`